pkg/northbound/gnmi/set.go

```diff
@@ -46,7 +47,7 @@ func (s *Server) Set(ctx context.Context, req *gnmi.SetRequest) (*gnmi.SetRespon
 		version          string // May be specified as 101 in extension
 		deviceType       string // May be specified as 102 in extension
 	)
-
+	var err error
```

Comment:
I do not believe you need a var here, just declare it inline underneath

Reply:
Cannot do inline because I'm doing **=** below not **:=**
I've moved it down to where it's needed to avoid any confusion with it